README.md

```diff
@@ -42,17 +47,19 @@ to configure, but up to you) to build a CI/CD over the service you have deployed
 
 ## Important Notes
 
-  * We recommend to try to run the application dockerized in your computer
+  * We recommend you to try to run the application dockerized locally on your computer
```

Comment:
Nope, `recommend you` is incorrect. It means you are the recommended one. Latin-like confusion.

Reply:
I partially disagree... but that's perfectly fine, I'm not a native English speaker :) 

But I'm pretty sure you can say:   `I recommend you (somthing/verb)`  -- without the **to** though. Let's have  aworkshop one day about that :rofl: